src/resources/views/fields/browse_multiple.blade.php

```diff
@@ -0,0 +1,174 @@
+@php
+$multiple = array_get($field, 'multiple', true);
+$value = old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : null));
+
+if (!$multiple && is_array($value)) {
+    $value = array_first($value);
+}
+@endphp
+
+<div @include('crud::inc.field_wrapper_attributes') >
+
+    <div><label>{!! $field['label'] !!}</label></div>
+    @include('crud::inc.field_translatable_icon')
+    @if ($multiple)
+        @foreach((array)$value as $v)
```

Comment:
Doesn't a `@foreach` imply an array (i.e. why the cast)?

Please rename `$v` to something semantic.

Reply:
Since I cannot be sure what the user has done, I assume the worst and cast any value to array (that way the field does not throw an error when the user "forgets" to enable proper field casting (or casts it to an object)).

I will update `$v` as soon as all other points have an answer, that way I can change everything at once.

Reply:
Errk, does PHP coerce objects to arrays if you use `(array)` (it's not something I've even attempted).